integrations/_infra/Dockerfile.base.runner

```diff
@@ -0,0 +1,12 @@
+ARG BASE_PYTHON_IMAGE=debian:trixie-slim
```

Comment:
can we choose specific python version? or atleast add a comment about what is the python version used here